### ISSUE TITLE
Lateral collateral ligament (LCL) tear at the elbow

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10797]
**Preferred term label:**
Lateral collateral ligament (LCL) tear at the elbow

**Synonyms**

**Definition (free text, please give PubMed ID)**
 ligamentous tear on the lateral aspect of the elbow; Lateral ulnar collateral ligament (LUCL) tear

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Radial head fracture
Capitellum fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001132
- label: Elbow lateral collateral ligament tear
- parents: [Abnormality of the elbow (HP:0009811)](https://hpo.jax.org/app/browse/term/HP:0009811)
- pmid: PMID:23261198
- def: An injury to the lateral collateral ligament (LCL) of the elbow. The LCL is a complex capsuloligamentous structure critical in stabilizing the ulnohumeral and radiocapitellar articulations. LCL injury can result in elbow instability, allowing the proximal radius and ulna to externally rotate away from the humerus as a supination stress is applied to the forearm.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10797
- synonyms: Lateral ulnar collateral ligament tear(NONE)|Ligamentous tear on the lateral aspect of the elbow(NONE)
